test/adapter/json_api/linked_test.rb

```diff
@@ -212,7 +212,7 @@ def test_underscore_model_namespace_for_linked_resource_type
             expected = {
               related: {
                 data: [{
-                  type: 'spam_unrelated_links',
+                  type: 'unrelated_links',
```

Comment:
why did the expectation change here?

aside, the type should probably be `spam:unrelated_links' or `spam-unrelated_links`

https://github.com/json-api/json-api/issues/850 (the issue I made that the jsonapi forum references )

(aside, why is there a `related=` writer on Post that is important?)